### PR TITLE
Change TFM to netcoreapp2.0

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AspNetCoreVersion>2.0.0-preview1-*</AspNetCoreVersion>
```
Comment (kichalla) [inline build/dependencies.props]: weird. This PR is on `dev` branch but the version here is `preview1`...something github is doing?

Comment (pranavkm) [inline build/dependencies.props]: Better? Set the wrong base

File: test/Microsoft.AspNetCore.Antiforgery.Test/Internal/DefaultClaimUidExtractorTest.cs [hunk 2]
```diff
@@ -60,13 +60,7 @@ public void DefaultUniqueClaimTypes_NotPresent_SerializesAllClaimTypes()
             identity.AddClaim(new Claim(ClaimTypes.Email, "someone@antifrogery.com"));
             identity.AddClaim(new Claim(ClaimTypes.GivenName, "some"));
             identity.AddClaim(new Claim(ClaimTypes.Surname, "one"));
-#if NET46
-            // CoreCLR doesn't support an 'empty' name
             identity.AddClaim(new Claim(ClaimTypes.NameIdentifier, string.Empty));
```
Comment (mikeharder) [inline test/Microsoft.AspNetCore.Antiforgery.Test/Internal/DefaultClaimUidExtractorTest.cs]: Is this now supported in netcoreapp2.0?

Comment (pranavkm) [inline test/Microsoft.AspNetCore.Antiforgery.Test/Internal/DefaultClaimUidExtractorTest.cs]: Yup, this works now. They removed a null or empty check in netstandard2.0

**Pull request merged**